Cargo.toml

```diff
@@ -19,7 +19,7 @@ members = [
     "quick-xml",
     "regex",
     "ring",
-    "rustfmt",
+    # "rustfmt",
```

Comment:
These targets are causing compilation issues for me. Going to disable temporarily to fix the build